lib/index.js

```diff
@@ -3340,6 +3340,55 @@ addMethod('take', function (n) {
     return s;
 });
 
+/**
+ * Creates a new Stream with the first values from the source as long as the function
+ * returns true or a truthy value
```

Comment:
```suggestion
 * returns true.
```

Let's not explicitly commit to handling truthy values.